### DISCUSSION TITLE
Alpine js, vite (replacing mix), and laravel

### DISCUSSION THREAD

**Discussion by Wotan6891** [tailpress/tailpress#135] [Ideas]
Hello Jeffrey van Rossum,
congratulations for this project.

I read on your website that you encouraged us to say ideas, so I say this:

Use Alpine js, vite (replacing mix), and laravel

in this way wordpress would remain as the administration panel.

It would be great since developing with wordpress everything that Laravel develops is very difficult.

note: i am a wordpress implementer, i can do everything using plugins and code overlays. I'm still not a pure code developer. and i have decided to learn laravel for its variety of advanced web development.

Greetings.

**Comment by jeffreyvr** [comment]
Hi!

> Use Alpine js, vite (replacing mix), and laravel

I'm thinking about Vite, but I have to experiment a bit more with it. Been using it in Laravel projects, and it's incredibly fast - yet I'm not sure what other benefits it would have in the context of a WordPress theme.

I've been using Alpine too, but as WordPress by default comes with jQuery - I tend go with that in most of my themes. Certainly for simple/minor things, I don't want to pull in yet another (be it small in Alpine's case) library. So not sure if if Alpine needs to be in TailPress by default.

Not sure what you mean to say with Laravel in this case.

> in this way wordpress would remain as the administration panel. 

I'm not sure what you mean by this. Feel free to explain further :)

**Reply by tareiking** [reply]
You might find this a really good step in the Vite + WP ecosystem: https://github.com/kucrut/vite-for-wp
